Tools/TestWebKitAPI/Tests/WebKitCocoa/WebsiteDataStoreCustomPaths.mm

```diff
@@ -1254,7 +1254,7 @@ HTTPServer server([&] (Connection connection) {
 
     auto websiteDataStoreConfiguration = adoptNS([[_WKWebsiteDataStoreConfiguration alloc] init]);
     websiteDataStoreConfiguration.get().generalStorageDirectory = generalStorageDirectory;
-    websiteDataStoreConfiguration.get().shouldUseCustomStoragePaths = false;
+    websiteDataStoreConfiguration.get().unifiedOriginStorageLevel = _WKUnifiedOriginStorageLevelBasic;
```

Comment:
I don't think we need to change the old tests since they only aim to check migration of IndexedDB and LocalStorage data. We could add a test to verify different behaviors on different values of _WKUnifiedOriginStorageLevelBasic  if necessary.